index.html

```diff
@@ -18,6 +18,10 @@
         <nav class="nav-bar">
           <a href="index.html" id="header-name">John Doe</a>   
           <i id="btn-menu" class="fa-solid fa-bars"></i>
+          <a href="#" class="nav-link">Portfolio</a>
+          <a href="#" class="nav-link">About</a>
+          <a href="#" class="nav-link">Contact</a>
```

Comment:
- [ ] Kindly use a `ul` `li` structure for the nav links for better semantics since it's a list 👍 
- [ ] To make the links functional, kindly add the `id` of the sections to the `href` attribute (if you don't have `id`s in the sections kindly add them) for example 👇 

```
<li><a href="#portfolio" class="nav-link">Portfolio</a></li>
```